__tests__/bin/vip-logs.js

```diff
@@ -65,9 +68,10 @@ describe( 'getLogs', () => {
 		expect( logsLib.getRecentLogs ).toHaveBeenCalledTimes( 1 );
 		expect( logsLib.getRecentLogs ).toHaveBeenCalledWith( 1, 3, 'app', 500 );
 
-		expect( console.log ).toHaveBeenCalledTimes( 2 );
-		expect( console.log ).toHaveBeenNthCalledWith( 1, '2021-11-05T20:18:36.234041811Z My container message 1' );
-		expect( console.log ).toHaveBeenNthCalledWith( 2, '2021-11-09T20:47:07.301221112Z My container message 2' );
+		expect( console.log ).toHaveBeenCalledTimes( 1 );
```

Comment:
What's going on here? Why isn't it logging the first line anymore?

Reply:
We now build an array and then only output once.

Reply:
Ah, sorry, missed the newline :)